Program.cs

```diff
@@ -26,18 +28,21 @@
         Environment.Exit(0);
     });
 
+threads = Environment.ProcessorCount;
+
 List<Prediction> errors = new();
 
+var sw = Stopwatch.StartNew();
+
 var (training, validation) = FileLoader.GetData("train.csv", offset, count);
 Console.Clear();
-Console.WriteLine("Data Load Complete...");
+Console.WriteLine($"Data Load Complete...{sw.ElapsedMilliseconds} ms");
 
 Classifier classifier = classifier_option
     switch
 {
-    "euclidean" => new EuclideanClassifier(training),
-    "manhattan" => new ManhattanClassifier(training),
-    _ => new EuclideanClassifier(training),
+    "manhattan" => new Classifier<ManhattanAlgo>("Manhattan Classifier", training),
+    "euclidean" or  _ => new Classifier<EuclideanAlgo>("Euclidean Classifier", training),
 };
 
 var start = DateTime.Now;
```

Comment:
Is this line `var start = DateTime.Now` still needed?

Reply:
@LeeGrissom I have replaced it with a Stopwatch.